languages/csharp/reference/README.md

```diff
@@ -8,89 +8,138 @@ The C# concept exercises are based on concepts. The list below contains the conc
 
 The core OO features a C# developer should know about are:
 
-- [Encapsulation](../../../reference/concepts/encapsulation.md)
-- [Classes](../../../reference/concepts/classes.md)
-- [Objects](../../../reference/concepts/objects.md)
-- [State](../../../reference/concepts/state.md)
-- [Mutation](../../../reference/concepts/mutation.md)
-- [Composition](../../../reference/concepts/composition.md)
-- [Inheritance](../../../reference/concepts/inheritance.md)
-- [Interfaces](../../../reference/concepts/interfaces.md)
-- [Polymorphism](../../../reference/concepts/polymorphism.md)
-- [Methods](../../../reference/concepts/methods.md)
+- [Classes][classes]
```

Comment:
How are these sorted currently? Does it make sense to sort them alphabetically?

Reply:
Good question. That is actually a great suggestion! I'll apply it.

Reply:
I've sorted the lists.